.licenserc.yaml

```diff
@@ -3,7 +3,7 @@ header:
     spdx-id: Apache-2.0
     copyright-owner: ByteDance
     content: |
-      Copyright 2022 Bytedance Ltd. and/or its affiliates.
```

Comment:
Should we use `2022-2023`?

Reply:
@BlockLiu Thank you for your reminder. I will fix it later

Reply:
@BlockLiu done